client/src/redux/prop-types.ts

```diff
@@ -50,6 +50,14 @@ export const UserPropType = PropTypes.shape({
       description: PropTypes.string
     })
   ),
+  savedChallenges: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      challengeFiles: PropTypes.array,
+      editableRegionBoundaries: PropTypes.array,
+      history: PropTypes.array
```

Comment:
This doesn't seem right. Both these should be properties of individual `challengeFile`s.